Tests/SwiftKueryORMTests/TestId.swift

```diff
@@ -110,7 +110,7 @@ class TestId: XCTestCase {
 
     struct IdentifiedPerson: Model {
         static var tableName = "People"
-        static var idKeypath: IDKeyPath = \IdentifiedPerson.id
+        static var idKeyPath: IDKeyPath = \IdentifiedPerson.id
```

Comment:
By breaking I mean like in this example.

Existing users of Kuery-ORM who have implemented a `Model` with an `idKeypath` property would find that the model no longer behaves as an identified model.  They would have to update their code and rename the property to match the new name.

Reply:
@djones6 Oh...
Now I cannot open my PC. So I will fix it and recommit later... Thank you! mm